src/raft/follower.go

```diff
@@ -315,11 +315,12 @@ func (r *Follower) startCheckUpgradeToC() {
 					}
 				}
 
-				if cnt >= r.GetQuorums() {
+				if cnt < r.GetQuorums() {
 					r.WARNING("ping.responses[%v].more.than.half.upgrade.to.candidate", cnt)
```

Comment:
The output does not match the actual meaning, move down to the outside of the block.